app/services/courses/content_status_service.rb

```diff
@@ -0,0 +1,23 @@
+module Courses
+  class ContentStatusService
+    def execute(enrichment, recruitment_cycle)
```

Comment:
I can't help but wonder if this shouldn't be simply parameterised on `course`. Why should the caller know that the `enrichment` and `recruitment_cycle` are required to calculate the content status? @danielburnley, did you have some thoughts on this?

Reply:
Yeah David and I did have a chat about it yesterday - This way prevents the service being directly coupled to how enrichments/recruitment cycles are stored on the course, more specifically it prevents us needing to pull in that "Last enrichments" logic into here which feels not particularly nice to have to extend to this service. 

I also prefer things exposing what they depend on specifically rather than giving them a catch-all, at least in the case of only a few arguments required. If we were closer to 5 args needing to be passed in I'd err on the side of the whole course

As an aside, I also prefer named arguments to provide some guidance for future callers and readability where called :eyes:

Reply:
Just chatted with @danielburnley IRL, and we agreed to leave it like this, with the intention to clean it up as part of the service containerisation changes. (https://github.com/DFE-Digital/manage-courses-backend/pull/949 et al)